.nprmc

```diff
@@ -0,0 +1,2 @@
+# .npmrc
+engine-strict=true
```

Comment:
Sorry I did not get to this sooner. Since both lines of the .nprmc file are removed through this and the previous conversation, that leaves .nprmc completely empty. Should I just remove the .nprmc file since there's nothing in it, leave it as is, or should I make some sort of change to it that keeps some kind of functionality?

Reply:
Correct, we don't need this file. It would be great if some of this stuff worked as advertised but NPM tends to fall short on a lot of that.